Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM registry.access.redhat.com/rhscl/ruby-23-rhel7
+
+# sshd
+USER root
+RUN yum install -y --setopt=tsflags=nodocs openssh-server libicu-devel && \
```

Comment:
@rhcarvalho I've got the exec form working now in the latest iteration. Let me know if that looks okay, or if you think it would be better to do it this way.